documentation/modules/configuring/con-config-kafka-connect-multiple-instances.adoc

```diff
@@ -3,7 +3,7 @@
 // assembly-kafka-connect-configuration.adoc
 
 [id='con-kafka-connect-multiple-instances-{context}']
-= Kafka Connect configuration for multiple instances
+= Configuring Kafka Connect for multiple instances
```

Comment:
Should this be in the deploying guide as well? It is in a way about configuring. But I think it is very relevant when you are deploying the Connect cluster because if you don't know about it, you will not succeed with the deployment.

Reply:
It's already in both guides